rslearn/train/tasks/per_pixel_regression.py

```diff
@@ -0,0 +1,257 @@
+"""Per-pixel regression task."""
+
+from typing import Any
+
+import numpy as np
+import numpy.typing as npt
+import torch
+import torchmetrics
+from torchmetrics import Metric, MetricCollection
+
+from rslearn.utils.feature import Feature
+
+from .task import BasicTask
+
+
+class PerPixelRegressionTask(BasicTask):
+    """A per-pixel regression task."""
+
+    def __init__(
+        self,
+        scale_factor: float = 1,
+        metric_mode: str = "mse",
```

Comment:
Because this only accepts a couple of values, we could do:


```suggestion
        metric_mode: Literal["mse", "l1"] = "mse",
```